spec/features/idv/flow_spec.rb

```diff
@@ -53,6 +53,19 @@
       expect(user.reload.active_profile).to be_a(Profile)
     end
 
+    scenario 'vendor agent throws exception' do
+      sign_in_and_2fa_user
+
+      visit verify_session_path
+
+      fill_out_idv_form_ok
+      fill_in 'profile_first_name', with: 'Fail'
```

Comment:
can we assign `Fail` to a variable that clarifies why this throws an exception? I assume this is one of those special values that triggers a failure but it took me a few reads of this test to come to that understanding